### ISSUE TITLE
Plugin does not load without WorldGuard

### ISSUE THREAD

**Issue by dalins** [Haaaqs/grave-chests#1]
Hi,

I use your plugin with following versions:

- Spigot: 1.12.2
- GraveChest: 1.2

And gravechest does not load with the following error:

```
[07:52:06 ERROR]: Could not load 'plugins/GraveChests-1.2.jar' in folder 'plugins'
org.bukkit.plugin.InvalidPluginException: java.lang.NoClassDefFoundError: com/sk89q/worldguard/protection/association/RegionAssociable
        at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:133) ~[spigot.1.12.2.jar:git-Spigot-2086bb0-2f3ed3b]
        at org.bukkit.plugin.SimplePluginManager.loadPlugin(SimplePluginManager.java:327) ~[spigot.1.12.2.jar:git-Spigot-2086bb0-2f3ed3b]
        at org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:248) [spigot.1.12.2.jar:git-Spigot-2086bb0-2f3ed3b]
        at org.bukkit.craftbukkit.v1_12_R1.CraftServer.loadPlugins(CraftServer.java:305) [spigot.1.12.2.jar:git-Spigot-2086bb0-2f3ed3b]
        at net.minecraft.server.v1_12_R1.DedicatedServer.init(DedicatedServer.java:205) [spigot.1.12.2.jar:git-Spigot-2086bb0-2f3ed3b]
        at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:545) [spigot.1.12.2.jar:git-Spigot-2086bb0-2f3ed3b]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_172]
Caused by: java.lang.NoClassDefFoundError: com/sk89q/worldguard/protection/association/RegionAssociable
        at java.lang.Class.forName0(Native Method) ~[?:1.8.0_172]
        at java.lang.Class.forName(Class.java:348) ~[?:1.8.0_172]
        at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:78) ~[spigot.1.12.2.jar:git-Spigot-2086bb0-2f3ed3b]
        at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:129) ~[spigot.1.12.2.jar:git-Spigot-2086bb0-2f3ed3b]
        ... 6 more
Caused by: java.lang.ClassNotFoundException: com.sk89q.worldguard.protection.association.RegionAssociable
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381) ~[?:1.8.0_172]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:152) ~[spigot.1.12.2.jar:git-Spigot-2086bb0-2f3ed3b]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:100) ~[spigot.1.12.2.jar:git-Spigot-2086bb0-2f3ed3b]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_172]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_172]
        at java.lang.Class.forName0(Native Method) ~[?:1.8.0_172]
        at java.lang.Class.forName(Class.java:348) ~[?:1.8.0_172]
        at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:78) ~[spigot.1.12.2.jar:git-Spigot-2086bb0-2f3ed3b]
        at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:129) ~[spigot.1.12.2.jar:git-Spigot-2086bb0-2f3ed3b]
        ... 6 more
```

I think an additional check against NULL in getWorldGuard() should fix it, but I am not so familiar with Java.

Best regards...

**Comment by Haaaqs** [comment]
Will take a look in a bit. The NULL check is pointless, the instanceof keyword works for NULLS also.

**Comment by Haaaqs** [comment]
issue fixed with release 1.3
